docs/source/blocks/block-style-wrapper.md

```diff
@@ -165,6 +167,54 @@ data like:
 
 will generate classnames `primary inverted`
 
+## Injecting custom CSS properties
+
+The style wrapper also allows to inject custom CSS properties, using the converter syntax.
+This use case is useful in some scenarios where the property that you are injecting is generic, customizable per project.
+
+```
+{
+  "styles": {
+    "backgroundColor:CSSProperty": "#222",
+  }
+}
+```
+
+will inject a style object prop in the component:
+
+```html
+<div class="block teaser" style="--background-color: #222">
+```
+
+```{note}
+Please notice that the resultant variable is transformed from camel case to kebab case.
```

Comment:
"Note" is inserted automatically in the admonition.

```suggestion
The resultant variable is transformed from camel case to kebab case.
```

Reply:
Removed, no longer applies.